src/main/java/api/apiwrappers/PetsApiWrapper.java

```diff
@@ -0,0 +1,61 @@
+package api.apiwrappers;
+
+import java.io.IOException;
+import api.CommonApiWrapper;
+import api.domain.pets.*;
+
+public class PetsApiWrapper extends CommonApiWrapper{
```

Comment:
Class name should be a noun, singular (`PetApiWrapper`). The endpoint name is also pet, not pets.

Reply:
I've refactored the code to reflect `PetApiWrapper`